docker-compose.example.yml

```diff
@@ -1,21 +1,25 @@
-version: '3.2'
-
-services:
-  mapjs:
-    build: .
-    restart: unless-stopped
-    working_dir: /app
-    command:
-      - sh
-      - -c
-      - |
-        npm install
-        npm start
-    volumes:
-      - .:/app
-      #- static:/static
-    ports:
-      - "8080:8080"
-
-#volumes:
-#  static:
\ No newline at end of file
+version: '3.2'
+
+services:
+  mapjs:
+    build: node:12
+    restart: unless-stopped
+    working_dir: /app
+    environment:
+      HEALTHCHECK_SECRET: "some_super_secret_value"
+    healthcheck:
+      test: "curl -fH \"Healthcheck-Secret: $HEALTHCHECK_SECRET\" http://localhost:8080 || exit 1"
+    command:
+      - sh
+      - -c
+      - |
+        npm install
```

Comment:
Believe it should be `npm run update` instead?

Reply:
I agree, we should use `npm run update` instead